system-int-test/test/cmr/system_int_test/search/granule_search_format_test.clj

```diff
@@ -47,6 +47,29 @@
           response (search/find-metadata :granule format-key params options)]
       (d/assert-metadata-results-match format-key [g1-echo] response))))
 
+(deftest search-smap-granule-with-size-in-echo10
```

Comment:
Do we have a test case for when `size` is `nil` or not in a format that we expect?

Reply:
1. We want to test the case when the iso map granule ingested contains transferSize, so the size is not going to be nil.
    If you really want, we could add a case when the transferSize doesn't exist, but that's like testing the current code, 
    not the code that I modified.
2. echo10 is the only retrieving format that we support right now when the original ingest format is iso smap.

Reply:
Fair enough - I'm just always in favor of being more defensive. Data isn't always perfect, so I want to make sure our code expects that to be the case, if that makes sense